orchagent/portsorch.cpp

```diff
@@ -3457,6 +3544,34 @@ bool PortsOrch::setDistributionOnLagMember(Port &lagMember, bool enableDistribut
     return true;
 }
 
+bool PortsOrch::addTunnel(string tunnel_alias, sai_object_id_t tunnel_id, bool hwlearning)
+{
+    SWSS_LOG_ENTER();
+
+    Port tunnel(tunnel_alias, Port::TUNNEL);
+    tunnel.m_tunnel_id = tunnel_id;
+    if(hwlearning)
+      tunnel.m_learn_mode = "hardware";
+    else
+      tunnel.m_learn_mode = "disable";
+    m_portList[tunnel_alias] = tunnel;
+    portOidToName[tunnel_id] = tunnel_alias;
```

Comment:
is this an unused variable?

Reply:
This is due to dependency on https://github.com/Azure/sonic-swss/pull/885 . Will check
on whether the dependency can be removed.